packages/vite/src/node/plugins/reporter.ts

```diff
@@ -66,6 +66,11 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
     )
   })
 
+  const isCompressibleFile = (fileName: string): boolean =>
+    ['html', 'json', 'svg', 'txt', 'xml', 'xhtml'].some((ext) =>
+      fileName.endsWith(`.${ext}`),
+    )
```

Comment:
nit: I'd prefer the available extensions be specified as regex instead like 

https://github.com/vitejs/vite/blob/ff02c9ace060ae9c9c654bc0ae1262597bb5c721/packages/vite/src/node/constants.ts#L49-L50

so we can run a simple `.test` call to check it.